Plugins/Json/src/JsonGeometryConverter.cpp

```diff
@@ -704,9 +702,9 @@ json Acts::JsonGeometryConverter::surfaceMaterialToJson(
       }
       smj[binkeys[ibin]] = binj;
     }
-    if (bUtility->transform() != nullptr) {
-      std::vector<double> transfo;
-      Acts::Transform3D transfo_matrix = *(bUtility->transform().get());
+    std::vector<double> transfo;
+    Acts::Transform3D transfo_matrix = bUtility->transform();
+    if (not transfo_matrix.isApprox(Acts::Transform3D::Identity())) {
```

Comment:
is this necessary to keep the json size manageable? otherwise I'd just write it out. Not too much of a fan of the isApprox method either, as it prevents bitwise equality of the json geometry and I haven't been able to figure out what EXACTLY the isApprox default (with no specified precision) does.

Reply:
The idea of this PR is not to change the behaviour, it only changes the memory model. I would rather leave it to @Corentin-Allaire as the main developer of the `json` writing module to decide, but I think we should see it in another PR.

Reply:
ok I assumed this was in fact changing behavior, but @Corentin-Allaire confirmed it wasn't. Good to go then.